### PR TITLE
adding node js program for json to PDF

### PR DESCRIPTION
**Pull request by praveen2896**
node js Program for converting json to PDF

### CODE REVIEW

File: playground/cli/jsonToPDF/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "fetch-json-data",
+  "version": "1.0.0",
+  "description": "convert json data to pdf",
+  "main": "\u001b[A\u001b[B",
```
Comment (johnjacobkenny) [inline playground/cli/jsonToPDF/package.json:5]: not sure what this means :)

File: playground/cli/jsonToPDF/script.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const fs = require('fs')
+const createHTML = require('create-html')
+const pdf = require('html-pdf')
+
+ 
+main();
+async function main(){
+    let data;
+    //getting the json file and then console
+    data= await require("./twit_thread.json");
```
Comment (johnjacobkenny) [inline playground/cli/jsonToPDF/script.js:11]: can refactor to use `require`?

**Review by johnjacobkenny — COMMENTED** [review]
Awesome!

File: playground/cli/jsonToPDF/package.json (hunk 1 above)
Comment (praveen2896) [reply playground/cli/jsonToPDF/package.json:5]: Typo mistake , let me correct and push again

File: playground/cli/jsonToPDF/script.js (hunk 2 above)
Comment (praveen2896) [reply playground/cli/jsonToPDF/script.js:11]: sure i will do @johnjacobkenny

**Comment by praveen2896** [comment]
typo mistakes fixed and code refactor also done, please someone take a review and suggest me

**Comment by tr0mbl3y** [comment]
@praveen2896 : twitter_thread.json is missing i think

**Comment by praveen2896** [comment]
hi @tr0mbl3y  it is already there please check , https://github.com/twindle-co/twindle/pull/364/files

**Comment by proful** [comment]
Screen shot of pdf output please

**Comment by praveen2896** [comment]
output

**Comment by praveen2896** [comment]
Hi @proful please find the screenshot of pdf in the previous comment

**Comment by proful** [comment]
Looking great

**Comment by praveen2896** [comment]
Thanks @proful

**Comment by proful** [comment]
You please review other javascript labeled or. Just give your suggestions

**Comment by praveen2896** [comment]
sure @proful

**Comment by johnjacobkenny** [comment]
@kingsley-einstein @tolgaerdonmez could you give your feedback on this :smiley:

**Comment by praveen2896** [comment]
output

File: playground/cli/jsonToPDF/script.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const Mustache = require("mustache"); 
+const fs = require('fs')
+const createHTML = require('create-html')
+const htmlToPdf = require('html-to-pdf');
+
+main();
+function main(){   
+    let data;
```
Comment (tolgaerdonmez) [inline playground/cli/jsonToPDF/script.js:8]: I believe you can use `const` for most of the variables here, but it isn't really important.

File: playground/cli/jsonToPDF/twit_thread.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Naval",
+    "twitterHandle": "naval",
+    "image": "https://pbs.twimg.com/profile_images/1256841238298292232/ycqwaMI2_400x400.jpg",
+    "createAt": "2018-05-31",
```
Comment (tolgaerdonmez) [inline playground/cli/jsonToPDF/twit_thread.json:6]: property `createAt` is converted to `createdAt` in pr #351 , can you also change them in your own code, to maintain a standart data.

**Review by tolgaerdonmez — CHANGES_REQUESTED** [review]
I think you could change a lot of `let` variables to `const`, it would be better for reading I think, lets hear what others say, its not a problem anyway 😄

**Comment by tolgaerdonmez** [comment]
@praveen2896 awesome work.

**Pull request merged**